conf/default.toml

```diff
@@ -26,6 +26,10 @@ proxy_path = ""
 # The default location in the dockerized datahub app for example is "/datahub/wc-embedder.html".
 # If the URL is not indicated, no permalinks will show up in the UI.
 # web_component_embedder_url = "/datahub/wc-embedder.html"
+# This optional parameter defines the languages that will be provided in a dropdown for the user to translate the UI.
```

Comment:
Having some padding above this block would make the config file more readable I think